examples/3_train_policy.py

```diff
@@ -85,7 +85,10 @@ def main():
     done = False
     while not done:
         for batch in dataloader:
-            batch = {k: v.to(device, non_blocking=True) for k, v in batch.items()}
+            batch = {
+                k: (v.to(device, non_blocking=True) if isinstance(v, torch.Tensor) else v)
+                for k, v in batch.items()
+            }
```

Comment:
In term of syntax, we should have a bias towards readability for beginners:
```python
batch = {}
for k, v in batch.items():
    if isinstance(v, torch.Tensor):
        v = v.to(device, non_blocking=True)
    batch[k] = v 
```
This principale is also consistent with `transformers`.

Reply:
Should we write `v = v.to(device, non_blocking=(device.type == "cuda"))` to avoid issues with `mps`?